### DISCUSSION TITLE
3.9.9.42编译一堆报错。。

### DISCUSSION THREAD

**Discussion by serviceqhd** [ttttupup/wxhelper#419] [Q&A]
![Uploading 企业微信截图_20240515165200.png…]()

**Comment by serviceqhd** [comment]
64-windows\include\fmt/core.h(1860): note: 查看对正在编译的函数 模板 实例化“fmt::v10::format_arg_store<fmt::v10::format_context,nlohmann::json_abi_v3_11_3::basic_json<std::map,std::vector,std::string,bool,int64_t,uint64_t,double,std::allocator,nlohmann::json_abi_v3_11_3::adl_serializer,std::vector<uint8_t,std::allocator<uint8_t>>,void>,nlohmann::json_abi_v3_11_3::basic_json<std::map,std::vector,std::string,bool,int64_t,uint64_t,double,std::allocator,nlohmann::json_abi_v3_11_3::adl_serializer,std::vector<uint8_t,sann::json_abi_v3_11_3::adl_serializer,std::vector<uint8_t,std::allocator<uint8_t>>,void>,nlohmann::json_abi_v3_11_3::basic_json<std::map,std::vector,std::string,bool,int64_t,uint64_t,double,std::allocator,nlohmann::json_abi_v3_11_3::adl_serializer,std::vector<uint8_t,std::allocator<uint8_t>>,void>>::format_arg_store<nlohmann::json_abi_v3_11_3::basic_json<std::map,std::vector,std::string,bool,int64_t,uint64_t,double,std::allocator,nlohmann::json_abi_v3_11_3::adl_serializer,std::vector<uint8_t,std::allocator<uint8_t>>,void>,nlohmann::json_abi_v3_11_3::basic_json<std::map,std::vector,std::string,bool,int64_t,uint64_t,double,std::allocator,nlohmann::json_abi_v3_11_3::adl_serializer,std::vector<uint8_t,std::allocator<uint8_t>>,void>>(nlohmann::json_abi_v3_11_3::basic_json<std::map,std::vector,std::string,bool,int64_t,uint64_t,double,std::allocator,nlohmann::json_abi_v3_11_3::adl_serializer,std::vector<uint8_t,std::allocator<uint8_t>>,void> &,nlohmann::json_abi_v3_11_3::basic_json<std::map,std::vector,std::string,bool,int64_t,uint64_t,double,std::allocator,nlohmann::json_abi_v3_11_3::adl_serializer,std::vector<uint8_t,std::allocator<uint8_t>>,void> &)”的引用
D:\vcpkg\installed\x64-windows\include\fmt/core.h(1842): note: 查看对正在编译的函数 模板 实例化“fmt::v10::detail::value<Context> fmt::v10::detail::make_arg<true,Context,nlohmann::json_abi_v3_11_3::basic_json<std::map,std::vector,std::string,bool,int64_t,uint64_t,double,std::allocator,nlohmann::json_abi_v3_11_3::adl_serializer,std::vector<uint8_t,std::allocator<uint8_t>>,void>,0>(T &)”的引用
        with
        [
            Context=fmt::v10::format_context,
            T=nlohmann::json_abi_v3_11_3::basic_json<std::map,std::vector,std::string,bool,int64_t,uint64_t,double,std::allocator,nlohmann::json_abi_v3_11_3::adl_serializer,std::vector<uint8_t,std::allocator<uint8_t>>,void>
        ]
D:\vcpkg\installed\x64-windows\include\fmt/core.h(1604): error C2338: static_assert failed: 'Cannot format an argument. To make type T formattable provide a formatter<T> specialization: https://fmt.dev/latest/api.html#udt'
[9/52] Building CXX object app\wxhelper\CMakeFiles\wxhelper.dir\src\db_controller.cc.obj
cl: 命令行 warning D9025 :正在重写“/MD”(用“/MDd”)
[10/52] Building CXX object app\wxhelper\CMakeFiles\wxhelper.dir\src\hook.cc.obj
cl: 命令行 warning D9025 :正在重写“/MD”(用“/MDd”)
[11/52] Building CXX object app\wxhelper\CMakeFiles\wxhelper.dir\src\contacts_controller.cc.obj
cl: 命令行 warning D9025 :正在重写“/MD”(用“/MDd”)
ninja: build stopped: subcommand failed.

**Reply by MyTkme** [reply]
我也编译出错，么有弄出来

**Comment by WIGIW1314** [comment]
我的也是一堆错误
![image](https://github.com/ttttupup/wxhelper/assets/168173068/2e6ff9b9-a0da-42e6-a039-3819c0c17fbd)

**Reply by serviceqhd** [reply]
一样一堆报错 不知道论坛里面那些人 编译通过的是咋通过的，，，，，

**Comment by ttttupup** [comment]
这种很基础的编译错误，可以直接问ai
